dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -2081,6 +2081,21 @@ def test_null_replication_metrics_calculation(self):
         np.testing.assert_array_almost_equal([[np.nan], [18]], column["class_sum"])
         np.testing.assert_array_almost_equal([[np.nan], [9]], column["class_mean"])
 
+        # Test with all null in a row
+        data_4 = pd.DataFrame(
+            [[10, 20], [9999999, 9999999], [30, 9999999], [9999999, 9999999]]
+        )
+
+        profiler = dp.StructuredProfiler(data_4, options=profile_options)
+        report = profiler.report()
+
+        self.assertTrue("null_replication_metrics" in report["data_stats"][0])
+        column = report["data_stats"][0]["null_replication_metrics"]
+
+        np.testing.assert_array_almost_equal([0.5, 0.5], column["class_prior"])
+        np.testing.assert_array_almost_equal([[20], [0]], column["class_sum"])
+        np.testing.assert_array_almost_equal([[10], [0]], column["class_mean"])
+
```

Comment:
If the entire dataframe is null, there is still an error.

Reply:
are these values the actual values we expect from the calculations or are these values just what was output from the `report` on L2090?

Reply:
They should be the actual values we expect

Reply:
yes

Reply:
excellent